internal/sync-pnpm/index.js

```diff
@@ -8,13 +8,16 @@ import { hardLinkDir } from '@pnpm/fs.hard-link-dir';
 import resolvePackagePath from 'resolve-package-path';
```

Comment:
The actual implementation of the watch mode for `sync-pnpm` is in this file... 

It's pretty straightforward at the moment but it might be worth pulling out into a separate repo and publishing since we want the same experience in `ember-headless-table` and probably `ember-toucan-core`...

Reply:
> might be worth pulling out into a separate repo and publishing

100%! I really like this idea.  I think using this in `ember-headless-table` + `ember-toucan-core` would be great for DX consistency across our OSS repos.  At the moment, this is all copied over in [toucan-core](https://github.com/CrowdStrike/ember-toucan-core/tree/main/internal/sync-pnpm), so having one source of truth would be huge.

> a separate repo

Do you think this should go under `@crowdstrike/` in npm for now, or not be namespaced as it's generally applicable to `pnpm`?

Reply:
The problem with putting it in a separate repo is that we'd have to think of a name for it :)

It is also a bit dependent on correctly setting up the scripts in your `package.json` files so I don't think it would be a good "off the shelf" experience for others as it is. 

I propose that we copy/paste it to the three places we need for now and learn a little about the required config and see if there is anything we can simplify. Then we can extract it along with docs or tools to help set it up right (could it lint your `package.json` files to make sure they'd work well?)...

Reply:
> When it comes to publishing - Do you think this should go under @crowdstrike/ in npm for now, or not be namespaced as it's generally applicable to pnpm?

When we do come to publishing, doing it under `@crowdstrike` would probably make sense as it makes maintenance going forward easier (since the org rather than an individual can own the keys that CI uses to publish)

Reply:
I was actually under the impression that there was a hard link between an npm scope and an npm organisation and so we would need to publish things under `@crowdstrike/` to maintain organisational ownership...

However, according to [this](https://docs.npmjs.com/about-organization-scopes-and-packages#managing-unscoped-packages) it seems like an org _can_ manage unscoped packages so it's fine :)